tools/c7n_azure/c7n_azure/provisioning/app_insights.py

```diff
@@ -0,0 +1,32 @@
+from msrestazure.azure_exceptions import CloudError
+
+from c7n_azure.provisioning.deployment_unit import DeploymentUnit
+from c7n_azure.provisioning.resource_group import ResourceGroupUnit
+
+
+class AppInsightsUnit(DeploymentUnit):
+
+    def __init__(self):
+        super().__init__('azure.mgmt.applicationinsights.ApplicationInsightsManagementClient')

```

Comment:
I'm sure the build caught this but in case you didn't know: Python 2 requires super to be called like this:
`super(AppInsightsUnit, self).__init__('azure.mgmt.applicationinsights.ApplicationInsightsManagementClient')`

Reply:
Yeah, noticed this :) Thanks!